website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+
+The new Kubernetes Contexts screen allows you to easily see all your registered Kubernetes contexts. You can use that screen to cleanup your registered contexts, and also to define the context you want to be using by default.
+
+![Kubernetes Contexts List](img/podman-desktop-release-1.6/kubernetes-contexts.png)
+
+
+### Editable Podman Machine 
+A Podman machine is a virtual environment specifically designed to run Podman containers on Mac and Windows. It allows users to manage and operate containerized applications in an isolated and controlled setting. When creating a Podman machine, you configure its settings: memory, CPU(s) and disk Size. 
+
+We've got feedback from some of you, interested by the ability to reconfigured your Podman machine on the go. This is now possible and particularly useful when you started with an environment and need to scale it up base on new needs and containers you would like to run in your Podman environment. 
+
+<ReactPlayer playing controls url="https://github.com/containers/podman-desktop/assets/1636769/91150767-58a9-47b5-abbc-58d2d50f4fca.gif" width='100%' height='100%' />
+
+You'll also notice, we improved the sliders to configure the Podman machine's options - and we also introduced a way to enter numeric values directly. 
+
+
+### Tabs/Filters for Containers and Pods
+Being able to quickly identify the containers and the pods you are working with is critical when you iterating on the development of your application. For this reason, we added filters at the top of the lists of Containers and Pods that allow you to easily see either see all the containers/pods, only those who are running or only those who are stopped.
+
+<ReactPlayer playing controls url="https://github.com/containers/podman-desktop/assets/1636769/0d20b5c2-517c-4ccc-8992-b8df275bcc30" width='100%' height='100%' />
+
+### Sorting for Volumes and Images lists
+The lists of Volumes and Images are getting improved and are now getting the ability to be sorted by the criteria of your choice. You can for example filter images by their size - which can be convenient when you want to clean up your environment. 
+
+<ReactPlayer playing controls url="https://github.com/containers/podman-desktop/assets/1636769/37190c74-7fa5-485e-81a4-bd970f606286" width='100%' height='100%' />
+
+### Environment columns on Containers and Pods lists
+Podman Desktop is able to work with multiple providers: it could work with multiple container engines and multiple kubernetes environments too. In order to make it easier to identify the containers and the pods and differentiate them depending on which environment they are running onto, we are introducing a new environment columns in the list of Containers and Pods to display a badge. 
```

Comment:
```suggestion
Podman Desktop is able to work with multiple providers: it could work with multiple container engines and multiple Kubernetes environments too. In order to make it easier to identify the containers and the pods and differentiate them depending on which environment they are running onto, we are introducing a new environment column in the list of Containers and Pods to display a badge. 
```